addin-proximity/app/scripts/main.js

```diff
@@ -256,15 +256,17 @@ geotab.addin.proximity = () => {
             let getLogRecord = results => new Promise((resolve) => {         
                 // if results have been limited, let the user know they may need to narrow search
                 if (results.length === maxLogRecordResults) {
-                    limitedDevices.push(encodeHTML(device.name));
+                    limitedDevices.push(encodeHTML(results[0].device.id));
```

Comment:
Is it meaningful to the user to see the list of device ID's that were limited?

Reply:
Yes, this is to let client have some idea that not all logs are being shown for a particular vehicle. And because the object does not have name of the vehicle and would need another API call (which might slow the addin), we stuck with the device IDs to give the information to the client.